tests/tests_oss_simple_flow.py

```diff
@@ -0,0 +1,159 @@
+import glob
+import os
+
+from mbdirector.benchmark import Benchmark
+from mbdirector.runner import RunConfig
+
+MEMTIER_BINARY = os.environ.get("MEMTIER_BINARY","memtier_benchmark")
+
+# run each test on different env
```

Comment:
Consider reorganizing this code for better reuse.

Reply:
split code into two files. One with the code parts expected to be reused across tests. Can you see if this was what you expected?